Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -32,4 +33,14 @@ public void setupCollectorCommands(
         oi.leftButtons.getifAvailable(1).whileHeld(eject);
         oi.leftButtons.getifAvailable(2).whileHeld(intake);
     }
+    
+    @Inject
+    public void setupClimbingCommands(
+            OperatorInterface oi,
+            DescendClimbingCommand descend,
+            ClimbClimbingCommand climb
+    )   {
+        oi.leftButtons.getifAvailable(3).whileHeld(descend);
```

Comment:
Question for whomever knows: what happens when both buttons are held? Does the latest-started one win?